.ruff.toml

```diff
@@ -49,7 +49,6 @@ select = [
     # mccabe ('C90')
 #    "C901",  # `{name}` is too complex ({complexity} > {max_complexity})
     # flake8-commas ('COM')
-    "COM812",  # Trailing comma missing
```

Comment:
Wait, why was this one removed?

Reply:
> Wait, why was this one removed?

Ruff format warns you that lint is incompatible

Reply:
```
$ ruff format .                                                          
warning: The following rules may cause conflicts when used with the formatter: `COM812`. To avoid unexpected behavior, we recommend disabling these rules, either by removing them from the `select` or `extend-select` configuration, or adding them to the `ignore` configuration.
2 files left unchanged
```

Reply:
Mmmh, but then, does it mean that ruff won't add the missing commas when needed? because it's one of the thing I always forget to add in multilines calls/defs...

Reply:
it's more subtle than that- ruff format adds trailing commas in most places that the rule would enforce, but not when a list of items does not run over a single line. On the other hand, COM812 enforces trailing commas *everywhere*

For example,

ruff format:
```python
def some_function(
    arg1, arg2, arg3, arg4
): -> None
```

vs

COM812:
```python
def some_function(
    arg1, arg2, arg3, arg4,
): -> None
```

Reply:
>  does it mean that ruff won't add the missing commas when needed?

I guess not, if the file is on the exclude list for formatting

I can add it back if you want?
I guess you will just have this warning showing, which isn't the end of the world

Reply:
> def some_function(
    arg1, arg2, arg3, arg4
): -> None

Oh ! I didn't know that this one was fine for flake8! I was worried about flake8 complaining actually so it's fine to remove the warning then!